### PR TITLE
[Server Process Manager] Convert TestPlan into Markdown

### PR DESCRIPTION
**Pull request by Jkat**
## Brief summary of changes
Convert Server Process Manager TestPlan into Markdown

#### Testing instructions (if applicable)

Look that it's in Markdown now

#### Links to related tickets (GitHub, Redmine, ...)

* Resolves https://github.com/aces/Loris/issues/4754

### CODE REVIEW

File: modules/server_processes_manager/test/TestPlan.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+1. Log in with a user that does not have the "View and manage server processes" permission. 
+   Check that when logged in there is no Sevrer Processes Manager sub menu in the Admin menu.
+   [Automation Test]
+2. Log in with a user that has the "View and manage server processes" permission. 
+   Check that there is a Server Processes Manager sub menu in the Admin menu.
+   [Automation Test]
+3. Check three filters, input correct data and click the show data button.Then check the table.
+   [Automation Test]
+4. Check the contents of the server processes page and remember the number of entries.
+   Make sure that the ImagingUploader Auto-Launch is set to Yes (Config module, Study section).
```
Comment (PapillonMcGill) [inline modules/server_processes_manager/test/TestPlan.md]:
```suggestion
   Ensure that the ImagingUploader Auto-Launch is set to Yes (Config module, Study section).
```

File: modules/server_processes_manager/test/TestPlan.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+   Using the Imaging uploader, upload an invalid scan (i.e. a `.tar.gz` file that is *not* a scan)
+   When the upload fails, verify that no new entrires were created in the server processes table.
```
Comment (PapillonMcGill) [inline modules/server_processes_manager/test/TestPlan.md]:
```suggestion
   When the upload fails, verify that no new entries were created in the server processes table.
```

File: modules/server_processes_manager/test/TestPlan.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+5. Upload a valid scan using the correct CandID, PSCID and Visit Label on the imaging_upload page.
+   After the progress bar reaches 100%, access the server processes manager page and check that a new
+   entry for the upload was created in the process tables. 
+   Using a Unix shell on the server on which the upload was done, do a tail -f on each of the process 
+   files listed in the table (stdout,stderr and exit code). Monitor the contents and make sure that they represent what
```
Comment (PapillonMcGill) [inline modules/server_processes_manager/test/TestPlan.md]:
```suggestion
   files listed in the table (stdout,stderr and exit code). Monitor the contents and ensure that they represent what 
```

Comment (PapillonMcGill) [inline modules/server_processes_manager/test/TestPlan.md]: which `table`?

Comment (Jkat) [reply modules/server_processes_manager/test/TestPlan.md]: @nicolasbrossard do you know which tables specifically?

Comment (PapillonMcGill) [reply modules/server_processes_manager/test/TestPlan.md]: @Jkat Would be nice to mention that the table is `server_processes` and `stdout`, `stderr`, and `exit_code_file` are attributes of that table.

Comment (nicolasbrossard) [reply modules/server_processes_manager/test/TestPlan.md]: The table that is referred to here is actually the search result table on the front-end, the one you see when you access the SPM page.

Comment (PapillonMcGill) [reply modules/server_processes_manager/test/TestPlan.md]: Ok thanks for the precision.

File: modules/server_processes_manager/test/TestPlan.md [hunk 4]
```diff
@@ -12,15 +11,15 @@ This test plan should be executed by someone with decent knowledge of the MRI up
...
-   Make sure that the ImagingUploader Auto-Launch is set to Yes (Config module, Study section).
-   Using the Imaging uploader, upload an invalid scan (i.e. a .tar.gz file that is *not* a scan)
-   When the upload fails, verify that no new entrires were created in the server processes table.
+   Ensure that the ImagingUploader Auto-Launch is set to Yes (Config module, Study section).
+   Using the Imaging uploader, upload an invalid scan (i.e. a `.tar.gz` file that is *not* a scan)
+   When the upload fails, verify that no new entries were created in the server processes table.
 5. Upload a valid scan using the correct CandID, PSCID and Visit Label on the imaging_upload page.
    After the progress bar reaches 100%, access the server processes manager page and check that a new
    entry for the upload was created in the process tables. 
    Using a Unix shell on the server on which the upload was done, do a tail -f on each of the process
```
Comment (PapillonMcGill) [inline modules/server_processes_manager/test/TestPlan.md]:
```suggestion
   Using a Unix shell on the server on which the upload was done, do a `tail -f` on each of the process 
```

**Comment by johnsaigle** [comment]
@driusan @PapillonMcGill Can you update your reviews?

**Pull request merged**